en/firefox/whatsnew/whatsnew-fx81.ftl

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/81.0/whatsnew/all/
+
+# html page title
+whatsnew81-page-title = What’s new with { -brand-name-firefox }
+
+# Main title
+whatsnew81-main-title = A new { -brand-name-firefox } for every device
+whatsnew81-main-intro = Whatever operating system you’ve got, there’s a better-than-ever, non-profit-backed mobile browser to put on it.
+whatsnew81-send-form-label = Enter your email for a download link
+whatsnew81-send-form-label-alt = Enter your email or phone number for a download link
+whatsnew81-scan-this-qr-code = Scan this QR code to download
+
+# content blocks below hero
+whatsnew81-the-new-firefox-for-android = The NEW { -brand-name-firefox } for { -brand-name-android } is here
+
+# "Weirdo" refers to features that are unusual in a delightful way. "Indie" is short slang for independent.
+whatsnew81-our-latest-for-android-has = Our latest for { -brand-name-android } has lots of fun, smart and even weirdo features – like a moveable search bar. Because we’re an indie browser. We care about your thumbs.
```

Comment:
- Punctuation for the last two sentences, should be separated by a comma, not a period and should read like this: `Because we’re an indie browser, we care about your thumbs.`
- I am beginning to wonder if it is a good practice to put a string that needs an explanation in the en-US file and provide a neutral English in the standard template.

Reply:
> I am beginning to wonder if it is a good practice to put a string that needs an explanation in the en-US file and provide a neutral English in the standard template.

It is a good practice to do that, and it was in our original plan. @alexgibson @craigcook thoughts?

Reply:
It was provided and approved as two sentences so I believe it's deliberate. I can ask about it though, and also ask for a more neutral alternative.